site/static/api/device-registry-v1.yaml

```diff
@@ -430,6 +430,11 @@ components:
                type: string
                description: A human readable error message of what went wrong.
 
+      Settings:
```

Comment:
*Settings* seems to suggest some specific semantics attached to the properties contained in it.
What about *Any* or *AnyProperties*?

Reply:
I am also not truly satisfied with "Settings". However part of this schema definition is the `description` and that gives a bit of explanation around this construct. So simply naming this "Any" wouldn't really fit with the description field anymore.

But I am open for suggestions to other names. Initially I had "Defaults".

Reply:
My understanding is that we use this type to contain arbitrary properties whose semantics is up to the user. If the only place we use this in is the *defaults* property of Tenant and Device, then *DefaultProperties* would not be bad. Otherwise maybe *ArbitraryProperties*?
The description should be adapted as well in any case.

Reply:
Ok, I changed this to `DefaultProperties` and I tweaked the description.